src/ui-kit/Link.tsx

```diff
@@ -1,12 +1,13 @@
 import React from "react";
 import { ILinkProps } from "./props.inteface";
+import {Link as RouterLink} from "react-router-dom";
 import Styles from "./Link.module.scss";
 
 const Link:React.FC<ILinkProps> = ({ children, url, target, style }: ILinkProps) => {
   return (
-    <a href={url} className={Styles.link} style={style} target={target}>
```

Comment:
instead of injecting style,it's better to inject className so it adds to the component's className and the output will be something like this:
<element className={ComponentClasses." ".injectedClasses} />
but if you inject style directly to the element it will cause inline-styles which is not best practice